File: Examples/DotNET/CSharp/Files/CopyFile.cs
using System;
using Com.Aspose.Storage.Api;
using Com.Aspose.Storage.Model;

namespace Files
{
    class CopyFile
    {
        public static void Run()
        {
            // ExStart:1           
            StorageApi storageApi = new StorageApi(Common.APP_KEY, Common.APP_SID, Common.BASEPATH);

            string Path = "testfile.txt";
            string newdest = "new-testfile.txt";            
            string versionId = null;
            string storage = null;
            string destStorage = null;        
            byte[] file = System.IO.File.ReadAllBytes(Common.GetDataDir() + Path);
            try
            {
                // Invoke Aspose.Storage Cloud SDK API to upload and copy file
                storageApi.PutCreate(Path, versionId, storage, file);
                ResponseMessage apiResponse = storageApi.PutCopy(Path, newdest, versionId, storage, destStorage,null);
                if (apiResponse != null && apiResponse.Status.Equals("Ok"))
                {
                    Console.WriteLine(apiResponse.Message);
                    Console.ReadKey();
                }
            }
            catch (Exception ex)
            {
                System.Diagnostics.Debug.WriteLine("error:" + ex.Message + "\n" + ex.StackTrace);
            }
            // ExEnd:1
        }
    }
}

File: Examples/PHP/Files/CopyFile.php
//ExStart:
<?php
require_once realpath(__DIR__ . '/..') . '/vendor/autoload.php';
require_once realpath(__DIR__ . '/..') . '/Utils.php';

use Aspose\Storage\StorageApi;
use Aspose\Storage\AsposeApp;

class Files {

	public $storageApi;

	public function __construct() {
		AsposeApp::$appSID = Utils::appSID;
		AsposeApp::$apiKey = Utils::apiKey;
		$this->storageApi = new StorageApi();
	}

	public function copyFile() {
		// Upload file to Aspose Cloud Storage
		$fileName = "testfile.txt";
		//Utils::uploadFile($fileName);

		$file = realpath(__DIR__ . '/../../..') . '/Data/' . $fileName;
        $result = $this->storageApi->PutCopy($Path=$fileName, $newdest="copy-testfile.txt", $versionId = null, $storage = null, $destStorage = null, $file);
        print_r($result);
	}
}

$files = new Files();
$files->copyFile();

?>
//ExEnd: